content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -246,6 +246,8 @@ be down for the same period as the previous example (`08:29:00` to `10:21:00`,)
 The CronJob is only responsible for creating Jobs that match its schedule, and
 the Job in turn is responsible for the management of the Pods it represents.
 
+Starting with Kubernetes v1.32 the job created by CronJob controller will have an annotation `batch.kubernetes.io/cronjob-scheduled-timestamp` which will show the original (expected) creation timestamp for a Job. The value of this field is in RFC3339 format.
```

Comment:
This is not technically true. Wouldn't this annotation have shown up in 1.28 and now we are forbidding users from turning it off in 1.32?

Reply:
Technically - yes, it should show up starting in 1.32, but since it was beta, it could potentially be disabled. Going with 1.32, where the feature is always on is safer assumption, b/c it will always appear. I thought for a while to go with 1.28, but based on what I wrote, I've decided to go with the stable.